rinja_derive/src/generator.rs

```diff
@@ -92,6 +96,7 @@ impl<'a> Generator<'a> {
             -> {CRATE}::Result<()> {{",
         ));
         buf.writeln(format_args!("use {CRATE}::filters::AutoEscape as _;"));
+        buf.writeln(format_args!("use std::fmt::Write;"));
```

Comment:
Please use `use ::core::fmt::Write as _;`. We already use `::core` for everything that is in `core`. The `as _` is useful not to shadow any other `Writer` name the user want's to use, e.g. `std::io::Writer`.

Reply:
:+1: